crates/parcel_plugin_rpc/src/nodejs/rpc_host_nodejs.rs

```diff
@@ -1,18 +1,26 @@
+use std::sync::mpsc::Receiver;
 use std::sync::Arc;
 
+use anyhow::anyhow;
+use parking_lot::Mutex;
+
 use crate::RpcHost;
 use crate::RpcWorkerRef;
 
-use super::worker_init::get_worker;
+use super::NodejsWorker;
 use super::NodejsWorkerFarm;
 
 pub struct RpcHostNodejs {
   node_workers: usize,
+  rx_worker: Arc<Mutex<Receiver<NodejsWorker>>>,
```

Comment:
why is there a lock around this?

Reply:
The `RpcHostNodejs` needs to be `Send + Sync`. I don't need the `Arc` though so I've removed that